JourneysSquad/JourneysSquad/View/BillRowView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+/// A view displaying information of the one bill.
+struct BillRowView: View {
+    /// The view model to provide data to the view.
+    @ObservedObject var viewModel: BillRowViewModel
+
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                Text(viewModel.description)
+                    .lineLimit(1)
+                    .font(.system(size: bodySize))
+
+                Text(viewModel.formattedParticipants)
+                    .lineLimit(1)
+                    .foregroundColor(.gray)
+                    .font(.system(size: noteSize))
+            }
+            Spacer()
+
+            Text(viewModel.sumOfBill)
+                .font(.system(size: bodySize))
+                .lineLimit(1)
+        }
+        .padding()
+    }
+    /// Font "Note" used as required [App style guidlines]
```

Comment:
```suggestion
    }
    
    /// Font "Note" used as required [App style guidlines]
```
It should be a line here.

Reply:
fixed